Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionImporter.cs

```diff
@@ -117,6 +131,76 @@ private static InputActionAsset CreateFromJson(AssetImportContext context)
             return asset;
         }
 
+        internal static void MigrateAllEnumParams(InputActionAsset asset)
+        {
+            foreach (var map in asset.actionMaps)
+            {
+                foreach (var action in map.actions)
+                {
+                    var raw = action.processors;
+                    if (string.IsNullOrEmpty(raw))
+                        continue;
+
+                    var rebuilt = new List<string>();
+                    foreach (var entry in raw.Split(';'))
+                    {
+                        var e = entry.Trim();
+                        if (e.Length == 0)
+                        {
+                            rebuilt.Add(e);
+                            continue;
+                        }
+
+                        var paren = e.IndexOf('(');
+                        string procName = paren >= 0 ? e.Substring(0, paren) : e;
+                        string args    = paren >= 0
+                            ? e.Substring(paren + 1, e.Length - paren - 2)
+                            : null;
+
+                        var procType = InputSystem.TryGetProcessor(procName);
+                        if (procType != null && args != null)
+                        {
+                            var patched = PatchEnumArgs(procType, args);
+                            rebuilt.Add($"{procName}({patched})");
+                        }
+                        else
+                        {
+                            rebuilt.Add(e);
+                        }
+                    }
+
+                    action.m_Processors = string.Join(";", rebuilt);
+                }
+            }
+        }
+
+        internal static string PatchEnumArgs(Type procType, string args)
```

Comment:
This should move to same place as InputActionAsset LoadFromJson for same resons as previous comment